roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -59,111 +187,123 @@ Output:
 
 ## Functionality
 
-This assumes a constant `DISPUTE_WINDOW: SessionWindowSize`. This should correspond to at least 1 day.
+This assumes a constant `DISPUTE_WINDOW: SessionWindowSize`. This should correspond to at least 1
+day.
 
 Ephemeral in-memory state:
 
 ```rust
 struct State {
-    keystore: KeyStore,
-    highest_session: SessionIndex,
+  keystore: Arc<LocalKeystore>,
+	rolling_session_window: RollingSessionWindow,
```

Comment:
nit: mixture of spaces and tabs
AFAIK, we're using spaces in the .md documentation elsewhere.